### DISCUSSION TITLE
customise Alarm_control_panel text on buttons displayed

### DISCUSSION THREAD

**Discussion by georgelza** [home-assistant/frontend#8207] [Configuration]
Hi,

I'd really like to customise the text shown on the buttons (arm away/arm home/arm night) so as to match them as per my alarm system (paradox integrated via PAI).

From what I can figure out the text is defined inside the alarm_control_panel.

 I've not been able to see if there is a way to do this, so thinking this might be a feature/option request.

George

**Comment by spacegaier** [comment]
Out of curiosity: What would those buttons then say in your case?

**Reply by georgelza** [reply]
the default options on InsiteGold is:
Arm
Stay 
Disarm
I'd make the options for us.
Arm
Stay 
Sleep
Disarm
but think rather than hard code it, allow the user top specify it, that way for the various systems the user can define it to align with their requirements.
G

**Reply by spacegaier** [reply]
I am not sure I understand what you want to achieve since the texts that we currently have are the same you want to set again?

The one you list as "Sleep" is apparently called "Night" in HA (just going by the docs, since I do not use any alarm in HA).

=> Not sure what you want to achieve here 😕

**Reply by georgelza** [reply]
want to make them read what I want them to read, to be the same as the vendor mobile app,
<img width="308" alt="Screen Shot 2021-01-23 at 10 10 03" src="https://user-images.githubusercontent.com/28488383/105573256-dd8c3c00-5d64-11eb-9592-f782fb5776f1.png">
 their text is currently fixed, not customisable.

Attached is a picture if what the buttons look like atm,
I want the 3 buttons to read.
"ARM"
"STAY"
"SLEEP"
G
